File: add/codesnippet/CSharp/m-system.convert.tostrin_23_1.cs
      sbyte[] numbers = { SByte.MinValue, -12, 17, SByte.MaxValue};
      System.Globalization.NumberFormatInfo nfi = new System.Globalization.NumberFormatInfo();
      nfi.NegativeSign = "~";
      nfi.PositiveSign = "!";
      foreach (sbyte number in numbers)
         Console.WriteLine(Convert.ToString(number, nfi));
      // The example displays the following output:
      //       ~128
      //       ~12
      //       17
      //       127      

File: add/codesnippet/VisualBasic/m-system.convert.tostrin_23_1.vb
      Dim numbers() As SByte = { SByte.MinValue, -12, 17, SByte.MaxValue}
      Dim nfi As New System.Globalization.NumberFormatInfo()
      nfi.NegativeSign = "~"
      nfi.PositiveSign = "!"
      For Each number As SByte In numbers
         Console.WriteLine(Convert.ToString(number, nfi))
      Next
      ' The example displays the following output:
      '       ~128
      '       ~12
      '       17
      '       127          